tests/test_validate_raw_file.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import tempfile
+import pytest
+import pandas as pd
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.validate_raw_file import validate_raw_file 
+
+def test_valid_file():
+    """Test with a valid .data file."""
+    with tempfile.NamedTemporaryFile(suffix=".data", delete=False) as valid_file:
```

Comment:
I don’t think we’re using tempfile for our other tests. You can just create the file and delete them in the conf cleanup file. Can you change it to that? So that its consistent with others?